js/components/contactDetails/contactDetails_controller.js

```diff
@@ -1,6 +1,13 @@
 app.controller('contactdetailsCtrl', ['ContactService', 'AddressBookService', 'vCardPropertiesService', '$routeParams', '$scope', function(ContactService, AddressBookService, vCardPropertiesService, $routeParams, $scope) {
 	var ctrl = this;
 
+	ctrl.show = false;
+
+	ctrl.clearContact = function() {
+		delete $routeParams.uid;
+		ctrl.show = false;
+	};
+
```

Comment:
TODO: Fix this, this was a feeble attempt at clearing the param from the URL. This should be triggered with ng-click on the back button.